### ISSUE TITLE
Check if fractions are made of auto-aligned compounds

### ISSUE THREAD

**Issue by RosaWagner** [fonttools/fontspector#436]
We noticed that enabling auto-alignment after converting from UFO to Glyphs often messes up with the precomposed fractions… because they are not done correctly.

- fractions needs to be made with numr / fraction sign / dnom (and not slash)
- positioning of each component can be adjusted with kerning between numr/fraction, and fraction/dnom. Fraction sign can have negative sidebearings to have less kerning pairs.
- In general, spacing and kerning of all numr/fraction/dnom should be done carefully to allow the fraction feature to display nice automatic fractions :)

**Comment by felipesanches** [comment]
All of Font Bakery's `"new check proposal"` issues are being either closed or migrated to the [Fontspector issue tracker](https://github.com/fonttools/fontspector/issues), as there's no intention to implement new checks on the Font Bakery codebase anymore.

If you believe this proposal is still valid, please post a message here asking for it to be migrated. Otherwise, on **July 20, 2025** (30 days from now) it will be closed due to inactivity.

**Comment by felipesanches** [comment]
The fontspector issue tracker does not currently use priority/importance labels, but when it was hosted at the fontbakery issue tracker:

* @RosaWagner applied "P4 Someday-maybe" on Jul 1, 2022
* @vv-monsalve applied "Severity 2" on Oct 28, 2022

**Comment by simoncozens** [comment]
I don't think this is actionable at binary level; it sounds like a Glyphs source check for auto-alignment. Also nobody has updated this since the issue was deprecated, so I'm closing.
